FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/transport/src/tls_freertos.c

```diff
@@ -124,13 +172,16 @@ static MbedtlsStatus_t tlsSetup( NetworkContext_t * pNetworkContext,
 
     if( mbedtlsError != 0 )
     {
-        LogError( ( "Failed to set default SSL configuration, error %d.", mbedtlsError ) );
+        LogError( ( "Failed to set default SSL configuration, with mbedTLS high level error %s"
```

Comment:
The log messages can avoid the text of "with mbedTLS high level error and with low level error" by just printing the error codes as this example https://github.com/aws/amazon-freertos/blob/master/libraries/freertos_plus/standard/tls/src/iot_tls.c#L239-L242

Reply:
Agree